Kickstarter-iOS/DataSources/SettingsAccountDataSource.swift

```diff
@@ -34,16 +34,16 @@ final class SettingsAccountDataSource: ValueCellDataSource {
                           inSection: SettingsAccountSectionType.payment.rawValue)
   }
 
-  func insertCurrencyPickerRow() -> IndexPath {
-    let cellValue = SettingsCellValue(user: nil, cellType: SettingsAccountCellType.currencyPicker)
+  func insertCurrencyPickerRow(currency: Currency) -> IndexPath {
```

Comment:
I was thinking about most of the method signatures that take `currency` to take unnamed parameter but last minute decided to be more explicit. Happy to change this to `methodName(_ currnecy: Currency)` 😎

Reply:
I think we'd probably want `insertCurrencyPickerRow(with currency: Currency)` but fine for now.

Reply:
Great minds think alike..and then go and change it to something lame...hahaha, used `with` in my first iteration :)